### ISSUE TITLE
Accept dynamic variables as values for actions

### ISSUE THREAD

**Issue by then3rdman** [bitfocus/companion-module-neodarque-stagetimer2#11]
Would be amazing if we could pass the dynamic variable of one module as as argument to the actions. Apparently it is fully supported but is implemented on a per module basis

**Comment by marklneumann** [comment]
I am posting to hopefully advance this request.  I have made a keypad that types into a custom variable to set a time for the timer.  But the "Set Timer Time" action does not accept variables as inputs even if the value of the variable is a valid input.
<img width="2168" alt="Screenshot 2023-10-21 at 5 24 20 PM" src="https://github.com/bitfocus/companion-module-neodarque-stagetimer2/assets/53580025/4ee266b4-bd8f-4719-9892-30c240ec7cf6">

**Comment by dnmeid** [comment]
You're correct, at the moment parsing of variables needs to be implemented by every module.
I just reference https://github.com/bitfocus/companion/issues/2345 here, which aims for implementing that functionality for all inputs.

**Comment by marklneumann** [comment]
Any updates on when this module might be updated to version 3.0 and have varriable support implemented?
